mindmeld/components/entity_resolver.py

```diff
@@ -567,33 +790,403 @@ def _predict_exact_match(self, entity):
                 item_value.pop("whitelist", None)
                 values.append(item_value)
 
-        return values
+        if len(values) < top_n:
+            logger.info(
+                "Retrieved only %d entity resolutions instead of asked number %d for "
+                "entity %r for type %r",
+                len(values), top_n, entity.text, entity.type,
+            )
 
-    def load(self):
-        """Loads the trained entity resolution model from disk."""
-        try:
-            if self._use_text_rel:
-                scoped_index_name = get_scoped_index_name(
-                    self._app_namespace, self._es_index_name
-                )
-                if not self._es_client.indices.exists(index=scoped_index_name):
-                    self.fit()
-            else:
-                self.fit()
+        return values[:top_n]
 
-        except EsConnectionError as e:
-            logger.error(
-                "Unable to connect to Elasticsearch: %s details: %s", e.error, e.info
+    def _load(self):
+        self.fit()
+
+
+class SentencebertCossimEntityResolver(EntityResolverBase):
```

Comment:
minor: SetenceBertCosSimEntityResolver?

Reply:
renamed to `SentenceBertCosSimEntityResolver`